src/Symfony/Bundle/FrameworkBundle/Resources/config/test.xml

```diff
@@ -22,8 +22,14 @@
 
         <service id="test.session.listener" class="Symfony\Component\HttpKernel\EventListener\TestSessionListener">
             <tag name="kernel.event_subscriber" />
-            <tag name="container.service_subscriber" id="session" />
-            <argument type="service" id="container" />
+            <argument type="service">
+                <service class="Symfony\Component\DependencyInjection\ServiceLocator">
+                    <tag name="container.service_locator" />
```

Comment:
That's an issue when using `findTaggedServiceIds`, but `ServiceLocatorTagPass` visits recursively all definitions by using `AbstractRecursivePass`, so it works :)